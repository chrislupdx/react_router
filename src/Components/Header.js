import React from 'react';
import { Link } from 'react-router-dom';

export default function Header() {
  return (
    <header>
      <nav>
        <Link to="/red">Red</Link>
        <Link to="/blue">blue</Link>
        <Link to="/yellow">yellow</Link>
      </nav>
    </header>
  );
}
