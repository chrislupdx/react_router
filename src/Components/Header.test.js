import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('header', () => {
  it('renders', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
})
;
