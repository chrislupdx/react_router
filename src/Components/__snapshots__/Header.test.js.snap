// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header renders 1`] = `
<header>
  <nav>
    <Link
      to="/red"
    >
      Red
    </Link>
    <Link
      to="/blue"
    >
      blue
    </Link>
    <Link
      to="/yellow"
    >
      yellow
    </Link>
  </nav>
</header>
`;
